packages/ng/src/app/formly/wrappers/error.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, Input, ViewChild, ViewContainerRef } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { FormlyFieldConfig, FieldWrapper, FormlyConfig } from 'ng-formly';
+import * as _ from 'underscore';

```

Comment:
Why underscore and not lodash ?
Anyway, prefer `import { chain } from 'underscore'` as it avoid including the whole library